graspologic/utils/utils.py

```diff
@@ -701,3 +701,79 @@ def plug_in_variance_estimator(x):
         return covariances
 
     return plug_in_variance_estimator
+
+
+def remove_vertices(graph, indices, return_vertices=False):
+    """
+    Remove vertices from an adjacency matrix.
+
+    Parameters
+    ----------
+    graph: np.ndarray, shape (n, n)
+        Adjacency matrix
+    indices: int or list
+        Index/indices of the vertex/vertices to be removed.
+    return_vertex: bool, optional
```

Comment:
name doesn't match param (but maybe we'll change anyway)

Reply:
fixed anyway for now in f60a358